templates/overrides/views/views-mini-pager.html.twig

```diff
@@ -5,14 +5,19 @@
  */
 #}
 {% if items %}
-  {% set previous =  items.previous ? {'label':  items.previous.text, 'path': items.previous.href } : {} %}
-  {% set next = items.next ? {'label':  items.next.text, 'path': items.next.href } : {} %}
-  {% set pages = {} %}
-  {% set pages = pages|merge([{ 'label': items.current, 'disabled': true }]) %}
   {{ pattern('pagination', {
-    items: pages,
-    alignment: 'center',
-    prev: previous,
-    next: next,
+    'alignment': 'center',
+    'prev': items.previous ? {
+      'label': items.previous.text,
+      'path': items.previous.href,
+    },
+    'items': [{
+      'label': items.current,
+      'disabled': true,
+    }],
+    'next': items.next ? {
+      'label': items.next.text,
+      'path': items.next.href,
+    },
```

Comment:
I wanted to add first/last links here, but then realized that we don't do that for views-mini-pager.
However, I still wanted to clean up the unnecessary local variables.